AgentAssistAngular/src/app/services/root.service.ts

```diff
@@ -352,7 +352,11 @@ export class RootService {
     let snippersArray = this.formatSnippetResponse(suggestions?.searchassist?.snippets || [],response)
     let filesArray = suggestions?.searchassist?.files || [];
     let searchResponse: any = {};
-    if(suggestions?.searchassist && Object.keys(suggestions?.searchassist)?.length == 1 && suggestions?.searchassist?.snippets && suggestions?.searchassist?.snippets?.length == 0){
+    if(suggestions?.searchassist && suggestions?.searchassist?.snippets && suggestions?.searchassist?.snippets?.length == 0){
+      delete suggestions.searchassist.snippets;
+    }
+
+    if(suggestions?.searchassist && Object.keys(suggestions?.searchassist)?.length == 1 && Object.keys(suggestions?.searchassist)[0] == 'searchRequestId'){
```

Comment:
@saradadh-Kore Object.keys(suggestions?.searchassist) assign this statement to a local variable and use that to make if condition looks simpler.

Reply:
made changes